modules/libcom/RTEMS/kernel/rtems_init.c

```diff
@@ -42,7 +42,7 @@
 #include "logClient.h"
 #include "osiUnistd.h"
 #include "iocsh.h"
-#include "osdTime.h"
+#include "../../src/osi/os/RTEMS-kernel/osdTime.h"
```

Comment:
It shouldn't be necessary to have this horrendous relative path here (anything else you see like this is a bad code smell and should be examined very closely). The appropriate version of osdTime.h should have been installed into a standard include directory before this gets built, so I'm sure this change isn't actually needed.